packages/experiments/src/components/PersonaCoin/PersonaCoinInitials/PersonaCoinInitials.tsx

```diff
@@ -1,13 +1,19 @@
 import * as React from 'react';
 import { Icon, Text } from 'office-ui-fabric-react';
+import { ISlotProp } from '../../../Foundation';
 import { IPersonaCoinProps } from '../PersonaCoin.types';
 import { getInitials, getRTL } from '../../../Utilities';
 
+export type IPersonaCoinInitialsSlot = ISlotProp<IPersonaCoinInitialsProps, string>;
+
+// TODO: Have Markus verify... className props was required, but I'm not sure if it should be.
+// If it is truly required, it seems the parent component should be ensuring that default values are
+// provided and not assume the consumer of the slot will provide them.
```

Comment:
@Markionium FYI, wasn't sure which way to go here. (Sorry for the typo on your name.)